docker-base-services.yml

```diff
@@ -38,6 +38,8 @@ services:
       - FLYWAY_BASELINE_ON_MIGRATE=true
       - FLYWAY_PLACEHOLDERS_api_db_username=fam_proxy_api
       - FLYWAY_PLACEHOLDERS_api_db_password=test
+      - FLYWAY_PLACEHOLDERS_admin_mgmt_api_db_user=fam_proxy_admin_mgmt_api
```

Comment:
Hi Ian, just wondering do we need to use mgmt? or we can just use management?

Reply:
Hi Catherine. Thanks. I simply chose the standard abbreviation for management "mgmt" for short and recognizable naming as a preference. I don't actually mind using "management". Do you like seeing it as "admin_management_api_db_user" for this variable instead? I can change it.